### PR TITLE
HubSpot CRM (Indpendent Publisher) - Fix scopes

### PR DESCRIPTION
**Pull request by troystaylor**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by vmanoharas** [comment]
Hello @troystaylor, 

We have seen that you submitted the PR, thank you very much for correcting it. 

We have noticed that you have submitted the PR, and we appreciate your effort in correcting it. However, we observed that you added scopes for only one action. It is essential to add scopes for all the actions to avoid unnecessary back and forth.

Therefore, we suggest creating an OAuth app using your account locally, testing it thoroughly, and then submitting the PR. This will ensure all necessary scopes are included and streamline the process.

Thanks for your patience and understanding.

**Comment by troystaylor** [comment]
@vmanoharas Please provide an update on why this PR is still open.

CC: @jopanchal @ShefaaliP

**Comment by vmanoharas** [comment]
Hello @troystaylor,

We have seen that you submitted the PR, thank you very much for correcting it.

We have noticed that you have submitted the PR, and we appreciate your effort in correcting it. However, we observed that you added scopes for only one action. It is essential to add scopes for all the actions to avoid unnecessary back and forth.

Therefore, we suggest creating an OAuth app using your account locally, testing it thoroughly, and then submitting the PR. This will ensure all necessary scopes are included and streamline the process.

Thanks for your patience and understanding.

**Comment by vmanoharas** [comment]
Additionally, this PR was submitted when there was an issue in the pipeline.  Please resubmit your PR/commit so that it can be triggered again and undergo validation? Thank you for your understanding. We apologize any inconvenience caused.

**Review by vmanoharas — COMMENTED** [review]
Hi @troystaylor 
Please see my comments. 
Additionally, this PR was submitted when there was an issue in the pipeline. Please resubmit your PR/commit so that it can be triggered again and undergo validation? Thank you for your understanding. We apologize any inconvenience caused.

File: independent-publisher-connectors/HubSpot CRM/apiProperties.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
           "identityProvider": "oauth2",
           "clientId": "[YOUR_VALUE]",
           "scopes": [
-            "crm.objects.contacts.read tickets crm.objects.marketing_events.read content actions timeline oauth forms files crm.objects.marketing_events.write crm.schemas.custom.read crm.objects.custom.read crm.objects.feedback_submissions.read crm.objects.custom.write"
+            "crm.objects.contacts.read tickets crm.objects.marketing_events.read content actions timeline oauth forms files crm.objects.marketing_events.write crm.schemas.custom.read crm.objects.custom.read crm.objects.feedback_submissions.read crm.objects.custom.write crm.objects.line_items.write"
           ],
           "redirectMode": "Global",
```
Comment (vmanoharas) [inline independent-publisher-connectors/HubSpot CRM/apiProperties.json]: Please modify redirectMode from "Global" to "GlobalPerConnector"

**Review by vmanoharas — COMMENTED** [review]
Hi @troystaylor 

Kindly resolve review comments.

**Review by vmanoharas — APPROVED** [review]
Hello @troystaylor ,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by vmanoharas** [comment]
[[certify-connector]]

**Comment by vmanoharas** [comment]
Hi @troystaylor, 

/crm/v3/objects/contacts/search action has been removed. Can you please confirm that no existing flows or users will be affected by this change?

**Pull request closed without merging**